packages/grafana-data/src/field/getFieldDisplayValuesProxy.ts

```diff
@@ -46,9 +46,8 @@ export function getFieldDisplayValuesProxy(options: {
         if (!field) {
           return undefined;
         }
-        // TODO: we could supply the field here for the getDisplayProcessor fallback but we would also need theme which
-        //  we do not have access to here
-        const displayProcessor = field.display ?? getDisplayProcessor();
+
+        const displayProcessor = field.display ?? getDisplayProcessor({field});
```

Comment:
Judging by comment on line 25 of display processor, theme should be optional and default to dark if not specified. I followed through with that so we can pass the field along for smarter defaults. I feel like this is a good change for grafana-data but not necessary for this PR. I can remove it.

Reply:
On second thought, I'm removing it and moving these changes into a separate PR https://github.com/grafana/grafana/pull/74421/files